library/std/src/os/windows/process.rs

```diff
@@ -125,6 +126,13 @@ pub trait CommandExt: Sealed {
     /// [2]: <https://msdn.microsoft.com/en-us/library/17w5ykft.aspx>
     #[unstable(feature = "windows_process_extensions_force_quotes", issue = "82227")]
     fn force_quotes(&mut self, enabled: bool) -> &mut process::Command;
+
+    /// Append literal text to the command line without any quoting or escaping.
+    ///
+    /// This is useful for passing arguments to `cmd.exe /c`, which doesn't follow
+    /// `CommandLineToArgvW` escaping rules.
+    #[unstable(feature = "windows_process_extensions_raw_arg", issue = "29494")]
+    fn raw_arg(&mut self, text_to_append_as_is: &OsStr) -> &mut process::Command;
```

Comment:
The signature of [`Command::arg`](https://doc.rust-lang.org/std/process/struct.Command.html#method.arg) uses the generic `AsRef<OsStr>` for the type. This should probably match that?

Reply:
I've added it. I'm not sure if `CommandExt` was supposed to be object-safe tho.